### DISCUSSION TITLE
开放地域列表

### DISCUSSION THREAD

**Discussion by feuyeux** [getlantern/lantern-forum-cn#176] [General]
目前app自动选择地域到HK等于没出去，ChatGPT,Bing,Gemini都封锁了HK，建议改进app能力，向付费用户提供动态可选的地域列表

**Comment by MaxMKMax** [comment]
@feuyeux 
你好。感谢您的反馈意见。是的，我同意你所说的。我们实际上计划通过让用户选择他们想要的服务器来升级应用程序，以允许他们使用最佳速度。一切都在进行中，请等待我们。谢谢。

**Reply by SimonHK** [reply]
什么是最佳速度，就这么HK，出都出不去要速度有毛用，总是HK怎么出去。用的太火大了。

**Comment by feuyeux** [comment]
还没解决，等待过程很漫长，置位open持续关注
